python/ray/data/_internal/execution/interfaces/physical_operator.py

```diff
@@ -435,3 +422,7 @@ def notify_in_task_submission_backpressure(self, in_backpressure: bool) -> None:
         if self._in_task_submission_backpressure != in_backpressure:
             self._metrics.on_toggle_task_submission_backpressure(in_backpressure)
             self._in_task_submission_backpressure = in_backpressure
+
+    def get_autoscaling_actor_pools(self) -> List[AutoscalingActorPool]:
+        """Return a list of `AutoscalingActorPool`s managed by this operator."""
+        return []
```

Comment:
Do we expect any operators other than `ActorPoolMapOperator` to override this method? Feel like it's not ideal to add a method to the `PhysicalOperator` interface just for one subclass, although I can't think of any alternatives off the top of my head.

Reply:
> use some top PhysicalOperator level attribute which describes if the operator relies on actors or tasks
I think this is redundant, because we can tell this by if get_autoscaling_actor_pools returns an empty list.